site/docs/master/output-file-format.md

```diff
@@ -97,3 +97,16 @@ resources/
                 ...
     ...
 ```
+
+## Output File Format Versioning
+
+The Velero output file format is intended to be relatively stable, but may change over time in order to support new features.
+
+In order to accommodate this, Velero follows [Semantic Versioning](http://semver.org/) for the file format version.
+
+Minor and patch versions will indicate backwards-compatible changes that previous versions of Velero can restore, including new directories or files.
+
+A major version would indicate that a version Velero older than the version that created the backup could restore it, usually because of moved or renamed directories or files.
```

Comment:
I think this is intended to say "A major version would indicate that a version **of** Velero older than the version that created the backup could **not** restore it, usually because of moved or renamed directories or files."

Reply:
Yep, will add these missing words :)